src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -5,73 +5,76 @@
       "Size": 3036
     },
     "classes.dex": {
-      "Size": 22484
+      "Size": 22488
     },
     "lib/arm64-v8a/lib__Microsoft.Android.Resource.Designer.dll.so": {
       "Size": 18288
     },
     "lib/arm64-v8a/lib_Java.Interop.dll.so": {
-      "Size": 86688
+      "Size": 88728
     },
     "lib/arm64-v8a/lib_Mono.Android.dll.so": {
-      "Size": 117712
+      "Size": 124072
     },
     "lib/arm64-v8a/lib_Mono.Android.Runtime.dll.so": {
-      "Size": 22384
+      "Size": 23432
     },
     "lib/arm64-v8a/lib_System.Console.dll.so": {
-      "Size": 24392
+      "Size": 24408
+    },
+    "lib/arm64-v8a/lib_System.Diagnostics.TraceSource.dll.so": {
+      "Size": 24056
```

Comment:
`dotnet new android` now appears to always include `System.Diagnostics.TraceSource.dll`. This app size regression test is for Mono (not CoreCLR), so how would that happen there?